async-enumerable-dotnet/AsyncEnumerable.cs

```diff
@@ -563,6 +563,28 @@ public static IEnumerable<T> ToEnumerable<T>(this IAsyncEnumerable<T> source)
             return new ToEnumerable<T>(source);
         }
 
+        /// <summary>
+        /// Converts the async sequence into a list.
+        /// </summary>
+        /// <param name="source">The source async sequence.</param>
+        /// <typeparam name="T">The element type.</typeparam>
+        /// <returns>The task returning the new List instance.</returns>
+        public static ValueTask<List<T>> ToList<T>(this IAsyncEnumerable<T> source)
```

Comment:
I'd like to reserve the no-postfix names to return IAsyncEnumerable. Task returning ones should either end in `Task` or `Async` if we want to establish `Async` as the convention.

Reply:
I'd also add a default `capacityHint` argument to both so that the list is not reallocated that often.

Reply:
Ok, I'll fix this.